docs/api.rst

```diff
@@ -36,11 +36,11 @@ Stream
     run :: Sink a -> Scheduler -> Disposable
   }
 
-A ``Stream`` represents a view of events over time. Its ``run`` method arranges events to be propagated to the provided :ref:`Sink` in the future. Each ``Stream`` has a local clock, defined by the provided :ref:`Scheduler`, which has methods for knowing the current time and scheduling future :ref:`Task`s.
+A ``Stream`` represents a view of events over time. Its ``run`` method arranges events to be propagated to the provided :ref:`Sink` in the future. Each ``Stream`` has a local clock, defined by the provided :ref:`Scheduler`, which has methods for knowing the current time and scheduling future :ref:`Tasks <Task>`.
 
-Some ``Stream``s, like :ref:`now`, are simple, while others, like :ref:`combine`, do sophisticated things such as combining multiple ``Stream``s or dealing with higher-order ``Stream``s.
+A ``Stream`` may be simple, like :ref:`now`, or may do sophisticated things such as :ref:`combining <combine>` multiple ``Stream`` s or deal with higher-order ``Stream`` s.
```

Comment:
Why the space after \`\`Stream\`\` and before s?

Reply:
Sphinx doesn't render it property if you omit the space.  The same is true for `:ref:`s.  The solution to `:ref:`s is to use the expanded form, but I haven't found a solution to the preformatted case.  The only option may be to reword the sentence to use singular instead of plural.

Reply:
_Sigh!_ So be it then.